code/script.js

```diff
@@ -1,47 +1,243 @@
 
-// Global variables, if you need any, declared here
+// Global variables
+let questionNumber = 1; 
+let globalFloor = "";
+let globalRoom = "";
+let globalMinutes = "";
+let globalName ="";
 
 // Functions declared here
+const nextQuestion = (message) => {
+  let question = ""; 
+  let user = "";          
+  if (questionNumber === 1) {
+    questionOne(message)
+  } else if (questionNumber === 2) { 
+    questionTwo(message);
+  } else if (questionNumber === 3) {
+    questionThree(message);
+  } else if (questionNumber === 4) {
+    questionFour(message);
+  } else if (questionNumber === 5) {
+    questionFive(message);
+  } 
+}
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
-  if (sender === 'user') {
+  soundEffect.play()
+  if (sender === 'user') {    //enda skillnaden mellan user och bot är css styling :) samma message-variabel
+    console.log(message)
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/noun_User_305942.png" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/noun_Chatbot_1596689.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello there, What's your name?`, 'bot') // "bot" & "user"
+}
+
+// Question 1
+const questionOne = (name) => { 
+  questionNumber++;
+  setTimeout(() => showMessage(`Hi ${globalName}, please select which floor you'd like your room on`, 'bot') ,2000)
+  setTimeout(() => makeButton(),2000)
+  const makeButton = () => {
+    inputWrapper.innerHTML = `
+    <button id="option-btn-1" class="option-btn" type="option" value="1">1</button>
+    <button id="option-btn-2" class="option-btn" type="option" value="2">2</button>
+    <button id="option-btn-3" class="option-btn" type="option" value="3">3</button>
+    `;
+    document.getElementById("option-btn-1").addEventListener("click", () => {
+      nextQuestion(document.getElementById("option-btn-1").value)
+    })
+    document.getElementById("option-btn-2").addEventListener("click", () => {
+      nextQuestion(document.getElementById("option-btn-2").value)
+    })
+    document.getElementById("option-btn-3").addEventListener("click", () => {
+      nextQuestion(document.getElementById("option-btn-3").value)
+    })
+  }
+}
```

Comment:
This code structure is repeated a few times what is the ideal situation to create a more generic function to handle this task every time 🤓 Please try to do it 😸 If you will you can always ask me for more tips 😸